rust/src/pgsql/parser.rs

```diff
@@ -2082,7 +2073,62 @@ mod tests {
             }
         }
 
-        // TODO keep adding more messages
+        //A series of response messages from the backend:
+        /* R   ·    S   ·application_name  S   ·client_encoding UTF8 S   ·DateStyle ISO, MDY S   &default_transaction_read_only off S   ·in_hot_standby off S   ·integer_datetimes on S   ·IntervalStyle postgres S   ·is_superuser off S   ·server_encoding UTF8 S   ·server_version 14.5 S   "session_authorization ctfpost S   #standard_conforming_strings on S   ·TimeZone Europe/Paris K      ···O··Z   ·I */
```

Comment:
formatting?

Reply:
/hides
VS Code made it look like it had line breaks. Am sorry for that >__<